pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+
+
+....................................................
+Before you submit a pull request, please make sure :
+
+- code contains tests relevant for the problem you are solving
+- all new and existing tests passed
+- all your commits have valid informative message
```

Comment:
You mean simply using the -m option in commit, or something more fancy?

Reply:
Besides the committs themselves, I would rather require that the description field of the pull request contains all the information needed: purpose, method (in short, and if not obvious as it could be the case in a few line changes PR), what changes are expected in the output (even none), what other PRs or externals it depends upon (if any), etc.